dashboard/app/models/course_offering.rb

```diff
@@ -384,6 +395,10 @@ def get_participant_audience
     course_versions&.first&.content_root&.participant_audience
   end
 
+  def get_instruction_type
```

Comment:
Good job figuring out the `content_root` stuff. Nit: A more rails-y method name would simply be `instruction_type`.